common/OpTestOpenBMC.py

```diff
@@ -429,14 +430,14 @@ def clear_sel_by_id(self):
             self.curl.run()
 
     '''
-    clear SEL : Clearing complete SEL is not yet implemented (TODO)
+    clear SEL : Clearing complete SEL
     curl -b cjar -k -H "Content-Type: application/json" -X POST \
     -d '{"data" : []}' \
-    https://bmc/org/openbmc/records/events/action/clear
+    https://bmc/xyz/openbmc_project/logging/action/DeleteAll
     '''
     def clear_sel(self):
         data = '\'{"data" : []}\''
-        obj = "/org/openbmc/records/events/action/clear"
+        obj = "/xyz/openbmc_project/logging/action/DeleteAll"
         try:
```

Comment:
Can you add here new verify_clear_sel function to check whether SEL has really zero entries or not, to make sure DeleteAll deleted every thing. You can call this fucntion in RestAPI testcase testcases/testRestAPI.py like this.
```
self.rest.clear_sel()
self.rest.verify_clear_sel()
```

Reply:
@ngorugan Can you add those both API's in Rest API testcase file  testcases/testRestAPI.py